src/plugins/filter/filter.js

```diff
@@ -38,6 +42,7 @@ var componentName = "wb-filter",
 			$elm, elmTagName, filterUI, prependUI,
 			settings, setDefault,
 			inptId, totalEntries;
+
```

Comment:
Remove this empty line

Reply:
@duboisp Are you sure it's wise to remove that empty line? Without it, the variable declarations will be directly followed by ``if`` on the next line (which looks kind of messy). Although FWIW that is the status quo...